FieldOpt/Reservoir/grid/cell.cpp

```diff
@@ -74,7 +79,10 @@ namespace Reservoir {
                 face.corners.push_back(corners_[face_indices_points[ii][1]]);
                 face.corners.push_back(corners_[face_indices_points[ii][2]]);
                 face.corners.push_back(corners_[face_indices_points[ii][3]]);
-                face.normal_vector = (face.corners[2] - face.corners[0]).cross(face.corners[1] - face.corners[0]).normalized();
+
+                auto diffv_a = (face.corners[2] - face.corners[0]);
```

Comment:
Creating temporary variables that are only used once is (in general) unnecessary. Here it would be better to just use linebreaks in the original code.